afnor690-2010-uco.csl

```diff
@@ -0,0 +1,628 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="never">
+  <!-- Standards AFNOR 690 Z 44-005 French adaptation 2010. Catholic University of the West Angers- Libraries -->
+  <info>
+    <title>AFNOR-690 édition 2010 (Université catholique de l'Ouest)</title>
+    <title-short>AFNOR-690 for UCO</title-short>
+    <id>http://www.zotero.org/styles/afnor690-2010-uco</id>
+    <link href="http://www.zotero.org/styles/afnor690-2010-uco" rel="self"/>
```

Comment:
Please add a "template" link for the style you started with to make this adaptation.